vispy/gloo/glir.py

```diff
@@ -876,18 +905,22 @@ def set_interpolation(self, min, mag):
 class GlirTexture2D(GlirTexture):
     _target = gl.GL_TEXTURE_2D
     
-    def set_size(self, shape, format):
+    def set_size(self, shape, format, internalformat):
```

Comment:
You mean the number of arguments of each GLIR command would be fixed, with possibly None values? If so, +1.